### ISSUE TITLE
Chat widget buttons lack visible focus indicator

### ISSUE THREAD

**Issue by arethsu** [JohnXLivingston/peertube-plugin-livechat#711]
**Description of issue**

This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 2.4.7 Focus Visible (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible).

Anyone who navigates with for example the tab key needs to know where the focus is in order not to get lost on the page.

The buttons inside the chat widget are all reachable by keyboard, but have no visible focus indicator.

![Emoji, members list, and send button highlighted to show which buttons lack a focus indicator, screenshot.](https://github.com/user-attachments/assets/4495735d-b67e-45d6-bb7e-9fe3f2ff6d7b)

This is because of the CSS rule in _buttons.scss that sets `outline: none` on the `:focus-visible` state.

![outline property highlighted in devtools, value is set to 0, screenshot.](https://github.com/user-attachments/assets/5f6e942c-b839-464c-ba26-a1fcc386870b)

**To Reproduce**

1. Use a normal keyboard and tab through the chat widget

**Recommended solution**

Avoid removing `outline` from elements that otherwise have it built-in. If you use correct HTML elements for all interactive components and do not change the outline with CSS, you usually meet this requirement.

On custom interactive components, you must ensure that there is an `outline` or other focus indicator around the component. Try using the keyboard to tab through the entire page - it should never be unclear where the focus is.

For even better accessibility, you can design your own outline to be a little wider and have better color contrast than the browser's default one, which is often light blue and difficult to see on bright pages. If you do this, the focus indicator should be at least 2px wide and have a color contrast of at least 3:1 against the background so that it's clearly visible. This is only a requirement at level AAA, but is a good guide to what a focus indicator should ideally look like. [Read more about focus indicators.](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance)

**Server:**

 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: TBA

**Desktop:**
 - OS: macOS 15.5
 - Browser: Safari 18.5

**Additional context**

This issue was part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), under assignment of [HowlRound Theatre Commons](https://howlround.com/).
